src/supremm/preprocessors/ProcPrometheus.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+""" Proc information pre-processor """
+
+import re
+
+from supremm.preprocessors.Proc import Proc
+from supremm.linuxhelpers import parsecpusallowed
+
+
+class ProcPrometheus(Proc):
+    """ Parse and analyse the proc information for a job. Supports parsing the cgroup information
+        from SLRUM and PBS/Torque (if available).
+    """
+
+    name = property(lambda x: "procprom")
```

Comment:
should be 'proc' not 'procprom'

see comment in the cpuusage plugin

Reply:
I have a fix for this in this commit https://github.com/ubccr/supremm/commit/7ac9f16c393045118491a9a9ccdadc678f1d148a